frontend/src/scenes/sessionRecordings/SessionPlayerDrawer.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { Drawer } from 'lib/components/Drawer'
+import { SessionsPlay } from 'scenes/sessions/SessionsPlay'
+import { useActions } from 'kea'
+import { ArrowTopLeftOutlined } from 'lib/components/icons'
+import { sessionRecordingsTableLogic } from './sessionRecordingsTableLogic'
+
+interface SessionPlayerDrawerProps {
+    personIds?: string[]
+    isPersonPage?: boolean
+}
+
+export function SessionPlayerDrawer({ personIds, isPersonPage = false }: SessionPlayerDrawerProps): JSX.Element {
+    const sessionRecordingsTableLogicInstance = sessionRecordingsTableLogic({ personIds })
+
+    const { closeSessionPlayer } = useActions(sessionRecordingsTableLogicInstance)
+    return (
+        <Drawer destroyOnClose visible width="100%" onClose={closeSessionPlayer}>
+            <>
+                <a onClick={closeSessionPlayer}>
+                    <ArrowTopLeftOutlined /> Back to {isPersonPage ? 'persons' : 'sessions'}
```

Comment:
I think with the separation of sessions and session recordings, it might make more sense to have 'sessions recordings' here than 'sessions'